R/aedseo.R

```diff
@@ -78,7 +83,8 @@ aedseo <- function(
       "quasipoisson"
       # TODO: #10 Include negative.binomial regressions. @telkamp7
     ),
-    na_fraction_allowed = 0.4) {
+    na_fraction_allowed = 0.4,
+    season = NULL) {
```

Comment:
Do you want to add a checkmate check for the season variable also?

```
checkmate::assert_integerish(season, len = 2, lower = 1, upper = 53, null.ok = TRUE)
```

Reply:
Thanks, I forgot it!